pywbemtools/pywbemcli/_cmd_class.py

```diff
@@ -823,58 +823,79 @@ def cmd_class_find(context, classname_glob, options):
         raise_pywbem_error_exception(er)
 
 
-def cmd_class_tree(context, classname, options):
+def get_class_hiearchy(conn, classname, namespace, superclasses=None):
     """
-    Execute the command to enumerate classes from the top or starting at the
-    classname argument. Then format the results to be displayed as a
-    left-justified tree using the asciitree library.
-    The --superclasses option determines if the superclass tree or the
-    subclass tree is displayed.
+    Get the class hiearchy from the server, either the superclasses
+    associated with a class or the subclasses.  If getting subclasses
+    the classname parameter may be None which requests that the complete
+    class hiearchy be retrieved..
+
+      Parameters:
+        conn(TODO):
+
+        classname(:term:`string):
+            classname if the tree is to be initiated from
+            within the class hiearchy. May be None
+
+        namespace(:term:`string):
+          Namespace to use to acquire classes from WBEM server
+
```

Comment:
Add a description of the `superclasses` parameter.

Reply:
Done

Reply:
DONE